Standards/scs-0217-v1-cluster-hardening.md

```diff
@@ -0,0 +1,470 @@
+  outside of Kubernetes.
+
+Most organizations and deployments work with RBAC, most often due to organizational or
+customer-owner-relationship-like structures in place.
+Nonetheless, neither ABAC, RBAC nor Webhook authorization can be recommended over the
+other, since this all depends on the use case and required structure of a deployment.
+Using at least one of these modes is recommended.
+
+It is also recommended to enable the Node authorizer in order to limit Kubelet
+permissions to a minimum operational state.
+
+#### Admission Controllers
+
+Admission controllers intercept requests to the Kubernetes API after the
+authentication and authorization step, which validate and/or mutate the request.
+This step is limited to `create`, `modify` and `delete` objects as well as custom
+verbs, but other requests are not blocked.
+Kubernetes provides multiple admission controllers, some of which are enabled by default.
+
+One recommended admission controller is the [`NodeRestriction` controller][node-restriction],
+which limits the `Node` and `Pod` objects a Kubelet is allowed to modify to their own `Node` or
+objects that are bound to them. It also disallows updating or removing taints and prevents changing
+or adding labels with a `node-restriction.kubernetes.io/` prefix.
+Be aware that Kubelets will only be limited by this admission controller, if the use credentials
+in the `system:nodes` group with a `system:node:<nodeName>` username. Administrators must therefore
+configure their Kubelets correctly, if the `NodeRestriction` controller should be fully functional.
+
+### Kubelet access control
+
+The Kubelet is the node agent that runs on each node. It registers with the API
+server and ensures, that pods handed over to it are running and healthy according
+to the specification provided to it. The HTTPS endpoint of a Kubelet exposes APIs
+with varying access to sensitive data and also enables various levels
+of performant operations enabling manipulation of node data and containers.
+There is also a read-only HTTP endpoint that was used for monitoring a Kubelet and
+its information. This port was also used by applications like `kubeadm` to check
+the health status of the Kubelet.
+This port is still available, but it is planned to be [removed][ro-port-removal]
+in a future version. At the moment, the port is disabled by default since [Kubernetes 1.10][ro-port-disabled]
+and shortly later also in [`kubeadm`][ro-port-disabled-kubeadm].
+Different sources recommend disabling this port [3][ro-port-s1] [4][ro-port-s2] due to possible
+security risks, but since this standard recommends restricting accessibility of internal ports,
+this port wouldn't be accessible from external networks.
+It is nevertheless recommended to keep this port disabled, since Kubernetes also acknowledged
+its risks and plans to remove it.
+
+By default, requests to the HTTPS endpoint that are not rejected by other authentication
```

Comment:
We can also mention `By default, the API server does not verify the kubelet's serving certificate` here - see https://kubernetes.io/docs/concepts/architecture/control-plane-node-communication/#api-server-to-kubelet

Reply:
Thanks for the catch, updated in [6805ea0](https://github.com/SovereignCloudStack/standards/pull/581/commits/6805ea0289d5595323c73ed5df8b29d9e90fd889)